setup.go

```diff
@@ -87,7 +85,10 @@ func parse(c *caddy.Controller) (*Metrics, error) {
 		if metrics != nil {
 			return nil, c.Err("prometheus: can only have one metrics module per server")
 		}
-		metrics = &Metrics{}
+		metrics = &Metrics{
```

Comment:
Maybe make a `New()` at some point, but meh.

Reply:
yeah, I had the same thought, but wanted to minimize merge conflicts with #35 😉

Reply:
On second thought... _there's no time like the present!_